openoa/utils/machine_learning_setup.py

```diff
@@ -190,7 +190,7 @@ def hyper_optimize(
             param_distributions=self.hyper_range,
             n_iter=n_iter_search,
             scoring=self.my_scorer,
-            verbose=1,
+            verbose=0,
```

Comment:
Would be preferable to be able to set this globally, or pass in the verbosity level.

Reply:
I agree, my one concern is that by using tqdm for tracking iterations alongside anything more granular than 0 produces the massive output that we had issues with in past versions. I'm going to add this to the logging issue unless there's a reason we should address it here and now.